src/material-experimental/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+package(default_visibility=["//visibility:public"])
+load("@angular//:index.bzl", "ng_module", "ng_package")
+load("//:packages.bzl", "CDK_TARGETS", "ROLLUP_GLOBALS")
+
+
+ng_module(
+  name = "material-experimental",
+  srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/material-experimental",
+  deps = [
+    "//src/lib:material",
```

Comment:
should this be `MATERIAL_TARGETS + CDK_TARGETS`?

Reply:
It's intentional; this prevents this package from using both styles of `@angular/material` imports